@here/harp-datasource-protocol/lib/Theme.ts

```diff
@@ -1003,6 +1003,14 @@ export interface TextStyleDefinition {
     vAlignment?: "Above" | "Center" | "Below";
 }
 
+/**
+ * Interface that defines a procedural atmosphere overlay effect.
+ */
+export interface Atmosphere {
```

Comment:
Is this used anywhere?

Reply:
This will be interface for between Theme (definition) and MapViewAtmosphere, but you have wright I will remove it from this PR for now.... Because Theme atmosphere definition (and parsing) is not implemented yet.

Reply:
I have removed it from this PR.